ctx.go

```diff
@@ -348,6 +356,20 @@ func (c *netHttpContext[B, P]) Body() (B, error) {
 	return body, err
 }
 
+func (c *netHttpContext[B, P]) Params() (P, error) {
+	var p P
```

Comment:
Assume this is purposely not implemented?

Reply:
In the next PR yes